StortSpelProjekt/Game/src/Gamefiles/Shop.h

```diff
@@ -40,9 +42,14 @@ class Shop
 	int GetPrice(std::string name);
 	// Get player currency balance
 	int GetPlayerBalance();
+	// Get upgrade image
+	Texture* GetUpgradeImage(std::string name);
```

Comment:
Why send a whole string object? wouldn't a constant string reference be good enough?